pulp_rpm/app/models/repository.py

```diff
@@ -272,6 +273,9 @@ def artifacts_for_version(version):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+        permissions = [
+            ("sync_rpmrepository", "Sync a repository"),
+        ]
```

Comment:
I am missing the "manage roles" permission. But maybe you want to add that in a separate commit.